cypress/e2e/anomaly.cy.js

```diff
@@ -99,6 +99,20 @@ describe("View and edit anomalies for a digital LPA", () => {
             ruleType: "empty",
             fieldOwnerUid: "certificate-provider",
           },
+          {
+            id: 130,
+            status: "detected",
+            fieldName: "address",
+            ruleType: "address-not-usable",
+            fieldOwnerUid: "attorney-1-uid",
+          },
+          {
+            id: 131,
+            status: "detected",
+            fieldName: "address",
+            ruleType: "address-not-usable",
```

Comment:
this doesn't impact the test but ideally the ruleType would be updated to invalidAddress...could you also update anomaly 129 😓

Reply:
Good find! This has been fixed, cheers.